acceptance/lib/helper.rb

```diff
@@ -61,7 +61,9 @@ def self.get_option_value(value, legal_values, description,
 
     # precedence is environment variable, option file, default value
     value = ((env_var_name && ENV[env_var_name]) || value || default_value)
-    if value
+    if value == ""
+      value = default_value
```

Comment:
@camlow325 I agree that the `get_option_value` behavior is confusing and I'm not sure what exactly can be done to make it better. We could add an optional argument to `get_option_value` so that it knows what the preferred type of its return value should be (`:string`, `:symbol`, etc)